README.rst

```diff
@@ -13,45 +13,36 @@ Beamline Simulation using `sirepo-bluesky`_ and Synchrotron Radiation Workshop (
 
 Purpose
 -------
-
-Use machine learning to simulate beamlines similar to SRW with lower computational costs.
-The goal is to determine whether or not a neural network can construct a beamline simulation given data from sirepo-bluesky.
+Use machine learning techniques to simulate beamlines given data from SRW.
 
 Installation
 ------------
 
 - Install `VirtualBox`_ on your computer.
 - Install `Vagrant`_ using the terminal.
 - Git clone this repository. Then use ``cd deep-beamline-simulation`` to move into the top level directory of this repository.
-- You will see a Vagrantfile containing virtual machine setup information. Start the virtual machine using ``vagrant up`` followed by ``vagrant ssh``.
-- Once the virtual machine is running, finish the Miniconda installation,
-.. code:: bash
+- You will see a Vagrantfile containing virtual machine setup information. Start the virtual machine using ``vagrant up`` followed by ``vagrant ssh``. To reload the virtual machine use ``vagrant reload`` and to update the machine for any changes use ``vagrant provision``.
```

Comment:
I would like to see some (but not all!) output from ``vagrant up`` and ``vagrant ssh`` in a code block, something like: (you could even use this)
```
% vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'bento/ubuntu-20.10' version '202107.28.0' is up to date...
==> default: Clearing any previously set forwarded ports...
==> default: Fixed port collision for 22 => 2222. Now on port 2201.

...

==> default: Machine already provisioned. Run `vagrant provision` or use the `--provision`
==> default: flag to force provisioning. Provisioners marked to run always will still run.
% vagrant ssh
Welcome to Ubuntu 20.10 (GNU/Linux 5.8.0-63-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

  System information as of Mon Dec 20 02:09:00 PM UTC 2021

  System load:  0.36               Users logged in:          0
  Usage of /:   58.6% of 30.88GB   IPv4 address for docker0: 172.17.0.1
  Memory usage: 30%                IPv4 address for eth0:    10.0.2.15
  Swap usage:   0%                 IPv4 address for eth1:    10.10.10.10
  Processes:    120


This system is built by the Bento project by Chef Software
More information can be found at https://github.com/chef/bento
Last login: Thu Dec  2 17:49:25 2021 from 10.0.2.2
vagrant@vagrant:~$
```
The reason is I would like these instructions to be approachable for people who have never used Vagrant.